test/instrumentation/modules/mysql/mysql.js

```diff
@@ -1,5 +1,6 @@
 'use strict'
 
+var host = process.env.MYSQL_HOST || 'localhost'
```

Comment:
No need to specify `localhost` as the fallback as it will happen automatically if `MYSQL_HOST` isn't set